### PR TITLE
fix test

### LINKED ISSUE #687: teal is failing checks
### What happened?

[A bug happened!](https://github.com/insightsengineering/teal/runs/7242767964?check_suite_focus=true)

### sessionInfo()

_No response_

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines.

### Security Policy

- [X] I agree to follow this project's Security Policy.

### PR DESCRIPTION
**Pull request by mhallal1**
fixes #687 

Same approach as https://github.com/insightsengineering/teal/pull/684/files

### CODE REVIEW

File: tests/testthat/test-get_rcode.R [hunk 1]
```diff
@@ -7,17 +7,21 @@ testthat::test_that("get_rcode returns header only for empty chunks", {
   # removing the Date line from the header as the seconds may be different
   # in the two strings
   testthat::expect_identical(
-    strsplit(r_code_from_chunks, "\n")[[1]][c(1:5, 7:14)],
-    strsplit(r_code_from_header, "\n")[[1]][c(1:5, 7:14)]
+    strsplit(r_code_from_chunks, "\n")[[1]][c(1:5, 7:15)],
+    strsplit(r_code_from_header, "\n")[[1]][c(1:5, 7:15)]
```
Comment (Polkas) [inline tests/testthat/test-get_rcode.R:11]: On my system should be up to 13 not 15

File: tests/testthat/test-get_rcode.R [hunk 2]
```diff
@@ -7,17 +7,21 @@ testthat::test_that("get_rcode returns header only for empty chunks", {
...
   teal.code::chunks_push(id = "test", chunks = ch, quote(a <- 1))
+
+  r_code_from_chunks <- get_rcode(chunks = ch)
+  r_code_from_header <- sprintf("\n\n%s\n\na <- 1", paste(get_rcode_header(), collapse = "\n"))
+
   testthat::expect_identical(
-    get_rcode(chunks = ch),
-    sprintf("\n\n%s\n\na <- 1", paste(get_rcode_header(), collapse = "\n"))
+    strsplit(r_code_from_chunks, "\n")[[1]][c(1:5, 7:17)],
+    strsplit(r_code_from_header, "\n")[[1]][c(1:5, 7:17)]
```
Comment (Polkas) [inline tests/testthat/test-get_rcode.R:24]: on my system should be up to 15 not 17

Comment (Polkas) [reply tests/testthat/test-get_rcode.R:24]:
```
> length(strsplit(r_code_from_chunks, "\n")[[1]])
[1] 15
> length(strsplit(r_code_from_header, "\n")[[1]])
[1] 15
```

File: tests/testthat/test-get_rcode.R (hunk 1 above)
Comment (mhallal1) [reply tests/testthat/test-get_rcode.R:11]: does it fail?

Comment (Polkas) [reply tests/testthat/test-get_rcode.R:11]: oh or some times 14

File: tests/testthat/test-get_rcode.R (hunk 2 above)
Comment (mhallal1) [reply tests/testthat/test-get_rcode.R]: updated the tests to use `length` so that it is consistent on any platform.

**Review by Polkas — APPROVED** [review]

**Pull request merged**